rmgpy/data/kinetics/family.py

```diff
@@ -2140,8 +2140,12 @@ def assignWeightsToEntries(entryNestedList, weightedEntries, N = 1):
             
             # Discard the last line, unless it's the only line!
             # The last line is 'Estimated using ... for rate rule (originalTemplate)'
-            if len(lines) == 1:
-                comment = lines[0]
+            #if from training reaction is in the first line append it to the end of the second line and skip the first line
+            if not 'Average of' in kinetics.comment:
+                if 'from training reaction' in lines[0]:
+                    comment = lines[1]+' '+lines[0]
```

Comment:
Do we need to do this, or could we just get rid of the line with 'from training reaction' since it doesn't seem like we use it?

Reply:
It gets looked for in retrieveOriginalEntry, which expects it to be on the same line as the primary comment.

Reply:
I don't think retrieveOriginalEntry uses it. If you look at line 2150 below, it splits the comment and takes the fourth token, which is the original template which was matched, which is the only argument given to retrieveOriginalEntry.

Reply:
Oh, you're right it grabs that information from the comment on the rule not the original kinetics comment even though it's present in both.

Reply:
This should be fixed now